### DISCUSSION TITLE
videos do not download as videos

### DISCUSSION THREAD

**Discussion by user4302** [Serene-Arc/bulk-downloader-for-reddit#452] [Q&A] [answered]
when i run 
python -m bdfr download D:/Downloads/BDFR --user me --saved --authenticate --file-scheme [{REDDITOR}]-[{DATE}]_{TITLE}_[{POSTID}]  --folder-scheme {SUBREDDIT} -L 1 --no-dupes

some videos download as an m4a audio file with the video's audio, and some as a 16kb gifv that doesnt play anything

Imgur links (that gave a video) seem to download as a 16kb gifv withy no content
and a reddit video downloads as an m4a audio file with the video's audio

**Comment by Serene-Arc** [comment, marked as answer]
Have you installed ffmpeg? it's a requirement for videos.

**Reply by user4302** [reply]
i did have it installed, but i reinstalled it (by following this guide https://www.wikihow.com/Install-FFmpeg-on-Windows)
and now i get an access denied when i type in ffmpeg -version

i think its a windows thing, ill figure this out and get back

**Reply by Serene-Arc** [reply]
Make sure it's in your PATH and that users have execution rights, then maybe reboot if necessary.

**Reply by user4302** [reply]
yeah users have "full control" of the FFMPEG folder, located in program files.
and it is added to my path.

**Reply by user4302** [reply]
![image](https://user-images.githubusercontent.com/32593782/121358034-e8c8b880-c94f-11eb-8748-d4fc81282924.png)
even if i add the folder to the desktop and "run as admin" i get this, which seems odd to me.

**Reply by user4302** [reply]
ok it was my antivirus sw...
it was blocking ffmpeg from running somehow.

so now that thats cleared. 
how do i associate the ffmpeg path to BDFR?
is it automatically linked when i ad the FFMPEG to the system env path?

**Reply by user4302** [reply]
everythign works now, thank you
